MetaMask/metamask-extension
# Part of #17670: Replace Typography with Text component

## Explanation

Updating `<Typography/>` to `<Text/>`  in `ui/pages/swaps/smart-transaction-status/smart-transaction-status.js`



## Screenshots/Screencaps















### Before

![Opera Snapshot_2023-04-13_155640_localhost](https://user-images.githubusercontent.com/68021378/231743879-873c0ef7-4b29-4718-9949-399c14b73f6d.png)



### After

![Opera Snapshot_2023-04-13_160842_localhost](https://user-images.githubusercontent.com/68021378/231744047-f198658f-ca29-43e5-84e0-61fdf79fe12f.png)




## Manual Testing Steps

- Go to the latest build of storybook on this PR
- Search `SmartTransactionStatusLabel` in the Side Panel
- Check It against develop storybook [https://metamask.github.io/metamask-storybook/?path=/story/pages-swaps-smarttransactionstatuslabel--smart-transaction-status-label-component](url)



## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
